simulator/src/resim/cmd_run.rs

```diff
@@ -13,6 +12,14 @@ pub struct Run {
     /// The path to a transaction manifest file
     path: PathBuf,
 
+    /// The network to use when outputting manifest, [simulator | adapanet | nebunet | mainnet]
+    #[clap(short, long)]
+    network: Option<String>,
+
+    /// Comma separated paths to blob files
+    #[clap(short, long)]
+    blobs: Option<String>,
```

Comment:
Minor: should we do `#[clap(multiple = true)]` or keep this as comme separated paths?

Reply:
Didn't know `multiple`. Seems to be better. Will update.